airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -122,15 +121,22 @@ class TIDeferredStatePayload(StrictBaseModel):
         ),
     ]
     classpath: str
-    trigger_kwargs: Annotated[dict[str, Any], Field(default_factory=dict)]
-    next_method: str
+    trigger_kwargs: Annotated[dict[str, Any] | str, Field(default_factory=dict)]
+    """
+    Kwargs to pass to the trigger constructor, either a plain dict or an ecnrypted string.
+
+    Both forms will be passed along to the trigger, the server will not handle either.
+    """
+
     trigger_timeout: timedelta | None = None
+    method_name: str
```

Comment:
Dont we want to continue calling this "next_method"?

Reply:
Yeah, that was my one question -- I think we do really.

Reply:
Yeah lets stick to it